### PR TITLE
1 add other sensors to the shield

### LINKED ISSUE #1: Add other sensors to the shield
The following sensors should be added:
- [x] Bosch Sensortec BNO055
- [ ] ST Microelectronics IIS2DLPC

### PR DESCRIPTION
**Pull request by andiaye33**

### CODE REVIEW

File: boards/shields/zest-sensor-imu/zest_sensor_imu.overlay [hunk 1]
```diff
@@ -17,4 +20,11 @@
...
 		reg = <0x76>;
 	};
-};
\ No newline at end of file
+	
+	iis2dlpc_zest_sensor_imu : iis2dlpc@33 {
+		compatible = "st,iis2dlpc";
+		reg = <0x33>; /* R19 */
+		drdy_gpios = <&sixtron_connector WKUP GPIO_ACTIVE_HIGH>; /* R18 and Rint2 */
+		drdy-int =  <1>; /* INT1*/
```
Comment (bperraudin) [inline boards/shields/zest-sensor-imu/zest_sensor_imu.overlay]: These lines should be removed as the interrupt pin is not connected by default.

File: boards/shields/zest-sensor-imu/zest_sensor_imu.overlay [hunk 2]
```diff
@@ -17,4 +20,11 @@
 		compatible = "bosch,bme280";
 		reg = <0x76>;
 	};
-};
\ No newline at end of file
+	
+	iis2dlpc_zest_sensor_imu : iis2dlpc@33 {
+		compatible = "st,iis2dlpc";
+		reg = <0x33>; /* R19 */
```
Comment (bperraudin) [inline boards/shields/zest-sensor-imu/zest_sensor_imu.overlay]: The comment should be removed, as reading the schematics is enough.

It allows not to have to keep this comment updated if we change the name of the resistor.

**Review by bperraudin — CHANGES_REQUESTED** [review]

File: boards/shields/zest-sensor-imu/zest_sensor_imu.overlay (hunk 1 above)
Comment (andiaye33) [reply boards/shields/zest-sensor-imu/zest_sensor_imu.overlay]: Resolved in [4dd5c7a](https://github.com/catie-aq/zephyr_zest-sensor-imu/commit/4dd5c7aebb82b512cc8260041e4f3889dd0f67a1).

File: boards/shields/zest-sensor-imu/zest_sensor_imu.overlay (hunk 2 above)
Comment (andiaye33) [reply boards/shields/zest-sensor-imu/zest_sensor_imu.overlay]: Resolved in : [4dd5c7a](https://github.com/catie-aq/zephyr_zest-sensor-imu/commit/4dd5c7aebb82b512cc8260041e4f3889dd0f67a1).

**Review by bperraudin — APPROVED** [review]

**Pull request merged**